index.html

```diff
@@ -516,996 +448,45 @@ <h2>Profiling Mechanism</h2>
-                <td>
-                  <span class="rfc2119-assertion" id="profile-5.1.7.2-thing-forms-additional-constraints-3">
-                  <code>scopes</code> MUST NOT be
-                  used.
-                  </span>
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </section>
-      </section>
-
-      <!-- Links -->
-
-      <section>
-        <h3 id="links">Links</h3>
-        <section class="ednote">TODO: Consider
-          selecting a defined set from [[RFC8288]].</section>
-
-        <p>
-          The "type" relationship as defined in chapter 6 of
-          [[RFC6903]] is reserved for indicating an instance
-          relationship between a thing and a thing template.
-          The <a>Core Data Model</a> does not put additional
-          constraints or requirements on links. The
-          interpretation of a link is out of scope.
-        </p>
-      </section>
-
-      <!--  Security -->
-
-      <section>
-        <h3 id="security">Security</h3>
-        <p>
-          <span class="rfc2119-assertion" id="profile-5.1.9-thing-security-1">
-          The <a>Core Data Model</a> defines a subset of the
-          security schemes that MAY be implemented on resource
-          constrained devices.
-          </span>
-          <span class="rfc2119-assertion" id="profile-5.1.9-thing-security-2">
-          A security scheme MUST be
-          defined at the thing level.
-          </span>
-          <span class="rfc2119-assertion" id="profile-5.1.9-thing-security-3">
-          The security scheme is
-          applied to the thing as a whole, a thing may adopt
-          multiple security schemes.
-          </span>
-        </p>
-        <p>
-          The set of security schemes supported in the <a>Core
-            Data Model</a> is based on the PlugFest results.
-            <span class="rfc2119-assertion" id="profile-5.1.9-thing-security-4">
-              To ensure interoperability, a TD consumer, which
-              compliant with the <a>Core Data Model</a> MUST
-              support <strong>all</strong> of the following security schemes:
-            </span>
-        </p>
-
-        <ul>
-          <li>no security</li>
-          <li>Basic Auth</li>
-          <li>Digest</li>
-          <li>Bearer Token</li>
-          <li>Oauth2</li>
-        </ul>
+    <p class="ednote">
+      The name "Core Profile" is still under discussion
+      (see <a href="https://github.com/w3c/wot-profile/issues/21">#21</a>).
+    </p>
 
-        <section>
-          <h4>Recommended Practice</h4>
-          <p>When using the "no security" or "Basic Auth"
-            security schemes it is strongly recommended to
-            use transport layer encryption.</p>
-        </section>
-      </section>
+    <!-- Identifier -->
+    <section id="identifier">
+      <h3>Identifier</h3>
+      <p class="ednote">
+        This section will define an identifier which can be included in a
```

Comment:
text should be added

Reply:
https://github.com/w3c/wot-profile/pull/117